scimax-functional-text.el

```diff
@@ -174,7 +174,7 @@ URL-PATTERN should have one %s in it which is replaced by username."
   "
 Open a @username
 _b_: Bitbucket  _f_: Facebook
-_g_: GitHUB     _i_: Instagram
+_g_: GitHub     _i_: Instagram
```

Comment:
This is kind of notable.

Normally I change `Github` to `GitHub`, but this is a bit different.